src/app/file-manager/services/register-download.service.ts

```diff
@@ -1,16 +1,22 @@
+
 import { Injectable } from '@angular/core';
 import { ElectronService } from 'ngx-electron';
 import { SecurityService } from './security.service';
 import { Item } from '../models/item';
 
 @Injectable()
 export class RegisterDownloadService {
-
   constructor(private electronService: ElectronService) { }
 
-  public startDownload(items: Item[]) {
+  public startDownload(items: Item[]): Promise<boolean> {
     if (this.electronService.isElectronApp) {
-      this.electronService.ipcRenderer.send('start-download', items, SecurityService.getAccessToken());
+      this.electronService.ipcRenderer.removeAllListeners('disk-space');
```

Comment:
What is this doing? I don't see `'disk-space'` used elsewhere in the code.

Reply:
This is now fixed.